apps/examples/e2e/tests/test-kbds.spec.ts

```diff
@@ -353,14 +360,6 @@ test.describe('Actions on shapes', () => {
 		// 	name: 'open-menu',
 		// 	data: { source: 'dialog' },
 		// })
-
-		/* --------------------- Export --------------------- */
-
-		await page.keyboard.press('Control+Shift+c')
-		expect(await page.evaluate(() => __tldraw_ui_event)).toMatchObject({
-			name: 'copy-as',
-			data: { format: 'svg', source: 'kbd' },
-		})
```

Comment:
We no longer emit an event when the canvas is empty, so I've moved this up to the `operations on shapes` section.